index.bs

```diff
@@ -691,183 +691,23 @@ it is not always *sufficient*
-it may be acceptable but beware that if the only time this happens is with BFCache,
-then it's likely many pages are not prepared for it. If it is not possible to support BFCache,
-follow the [[#discard]] pattern described below.
-
-Additionally, when a document becomes [=Document/fully active=] again,
-it can be useful to update it with the current state of the world,
-if anything has changed while it is in the non-[=Document/fully active=] state.
-However, care needs to be taken with events that occurred while in the BFCache.
-When not [=Document/fully active=], for some cases, all events should be dropped,
-in some the latest state should be delivered in a single event,
-in others it may be appropriate to queue events or deliver a combined event.
-The correct approach is case by case and should consider privacy,
-correctness, performance and ergonomics.
-
-Note: Making sure the latest state is sent to a document that becomes
-[=Document/fully active=] again is especially important when retrofitting existing APIs.
-This is because current users of these APIs expect to always have the latest information.
-Dropping state updates can leave the document with stale information,
-which can lead to unexpected and hard-to-detect breakage of existing sites.
+Explicitly specify how your feature behaves in non-[=Document/fully active=] BFCached documents when necessary.
 
-<div class="example">
-  The {{gamepadconnected}} event
-  can be sent to a document that becomes [=Document/fully active=] again
-  if a gamepad is connected while the document is not [=Document/fully active=].
-  If the gamepad was repeatedly connected and disconnected,
-  only the final connected event should be delivered.
-  (This is not specified yet, see [issue](https://github.com/w3c/gamepad/issues/149))
-</div>
-<div class="example">
-  For geolocation or other physical sensors,
-  no information about what happened while not [=Document/fully active=] should be delivered.
-  The events should simply resume from when the document became [=Document/fully active=].
-  However, these APIs should check the state when the document becomes [=Document/fully active=] again,
-  to determine if a status update should be sent (e.g. is the current location far away from the
-  location when the document becomes not fully active?), to ensure the document has the latest
-  information, as guaranteed by the API normally.
-</div>
-<div class="example">
-  For network connections or streams,
-  the data received while not [=Document/fully active=] should be delivered only
-  when the document becomes [=Document/fully active=] again,
-  but whereas a stream might have created many events with a small amount of data each,
-  it could be delivered as smaller number of events with more data in each.
-</div>
+If your API does things that fall into **any** of the below categories:
+* Interacts with a document from the "outside" (e.g. sends information to a document)
```

Comment:
```suggestion
* Interacts with a document from the "outside"
  (e.g. sends information to a document)
```

Reply:
Maybe "Interacts with an external document" could be better? Quotes tend to implicitly suggest that the quoted term is pseudo-something.